dts/bindings/iio/adc/intel,qmsi-adc.yaml

```diff
@@ -0,0 +1,35 @@
+#
+# Copyright (c) 2018, Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+---
+title: Intel ADC
+id: intel,qmsi-adc
+version: 0.1
+
+description: >
+    This is a representation of the intel ADC node
+
+inherits:
+    !include adc.yaml
+
+properties:
+    compatible:
+      type: string
+      category: required
+      description: compatible strings
+      constraint: "intel,qmsi-adc"
```

Comment:
not intel and not qmsi, this is designware

Reply:
ok